network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -55,6 +58,36 @@ def generate_image_field():
     })
 
 
+def generate_image_text_field():
+    image_id = choice(Image.objects.all()).id
+    image_text = fake.paragraph(nb_sentences=1, variable_nb_sentences=False)
+    url = fake.url(schemes=['https'])
+    alt_text = ' '.join(fake.words(nb=5))
+    top_divider = fake.boolean()
+    bottom_divider = top_divider
```

Comment:
`bottom_divider = top_divider` as realistically speaking top and bottom dividers exist as pair.

Reply:
Seems reasonable, although as long as the seed is fixed, the fact that they're different values is less important than the fact that they're "the same value each time"